src/utiles/getContentForUser.ts

```diff
@@ -54,42 +51,5 @@ export async function getPurchases(email: string) {
     };
   });
 
-  if (LOCAL_CMS_PROVIDER) {
-    return courses;
-  }
-
-  const baseUrl = `${APPX_BASE_API}/get/checkemailforpurchase`;
```

Comment:
Removed the entire appx dependency from the project

Reply:
hmm this might be tricky we need some backward compatability for the current courses

Reply:
yep that was what I was basically worried about but seems doable. 
I am here to put every ounce of effort for this pr just let me know!!!!